packages/core/src/components/axes/zoom-bar.ts

```diff
@@ -56,16 +56,17 @@ export class ZoomBar extends Component {
 
 	render(animate = true) {
 		const svg = this.getContainerSVG();
-		const options = this.model.getOptions();
 
-		const isDataLoading = Tools.getProperty(options, "data", "loading");
+		const isTopZoomBarLoading = this.services.zoom.isZoomBarLoading("top");
+		const isTopZoomBarLocked = this.services.zoom.isZoomBarLocked("top");
```

Comment:
This should be using an ENUM for the position, `AxisPositions.TOP`...

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/776/commits/f9d15696608945a99fa5b1fb7255dee7bf2ba634.